content/departments/cloud/technical-docs/index.md

```diff
@@ -111,7 +111,9 @@ All metrics can be seen in [scoped projects dashboard](https://console.cloud.goo
 
 Every customer managed instance has alerts configured:
 
-- [uptime check - version v1.1](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/main/modules/terraform-managed-instance-new/infrastructure.tf#L529) configured in dedicated GCP managed instance project
+- cloud provider-managed uptime check is configured in dedicated GCP managed instance project
+  - [v1.1](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/main/modules/terraform-managed-instance-new/infrastructure.tf#L529)
+  - [v2.0](https://github.com/sourcegraph/controller/blob/0091a3b6fdad81297580499f26764befb7b72d21/internal/resource/monitoring/monitoring.go#L76-L114)
```

Comment:
The first link is not a permalink... in fact I think the line it links to is already outdated. Could you update with an updated permalink? @michaellzc

Reply:
https://github.com/sourcegraph/handbook/pull/5775 done